go/otgrpc/client.go

```diff
@@ -81,3 +72,167 @@ func OpenTracingClientInterceptor(tracer opentracing.Tracer, optFuncs ...Option)
 		return err
 	}
 }
+
+// OpenTracingStreamClientInterceptor returns a grpc.StreamClientInterceptor suitable
```

Comment:
Please clarify that the instrumentation creates a single Span for the entire streaming request (not one Span per streaming response).